.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# https://github.com/changesets/action
+name: Release
+
+on:
+  push:
+    branches:
+      - phase21
+
+env:
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
# This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits

This is the comment on changeset documents